packages/babel-plugin/src/__tests__/index.test.ts

```diff
@@ -201,4 +201,166 @@ describe('babel plugin', () => {
 
     expect(actual).toInclude('c_MyDiv');
   });
+
```

Comment:
Do we still have cases where compression is not safe, hence we should bail out?

Reply:
Compression should be safe everywhere now.